sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -2738,12 +2785,14 @@ def set_datapath_deinit(self, channel):
                 data &= ~(1 << lane)
         self.xcvr_eeprom.write(consts.DATAPATH_DEINIT_FIELD, data)
 
+    @read_only_cache_dict_and_list
```

Comment:
not sure we can cache this, this looks to me a state

Reply:
Agree, I'll remove this one.